docs/design.html

```diff
@@ -534,6 +534,11 @@ <h4><a id="design_compactionconfig" href="#design_compactionconfig">Configuring
     being written to. The active segment will not be compacted even if all of its messages are older than the minimum compaction time lag.
     </p>
     <p>
+    The log cleaner does not clean the active log segment. Therefore pick a suitable time to roll the segment file when using compaction.</p>
+    <pre class="brush: text;">  segment.ms</pre>
```

Comment:
It's a little odd to mention only `segment.ms`. Shouldn't we mention `log.segment.bytes` as well? For example:

> The log cleaner does not clean the active log segment. You should take this into account when choosing the log rolling policy for the topic. For example, you can tune  `log.roll.ms` to roll the segment more frequently, or you can use `log.segment.bytes` to limit the maximum size of the active segment.

Best if we can link to these configs as well.